### PR TITLE
Feature/828 create missing transfer connection radiogram component

### PR DESCRIPTION
**Pull request by Greenscreen23**
### PR Checklist

Please make sure to fulfil the following conditions before marking this PR ready for review:

-   [ ] ~If this PR adds or changes features or fixes bugs, this has been added to the changelog~
-   [ ] ~If this PR adds new actions or other ways to alter the state, [test scenarios](https://github.com/hpi-sam/digital-fuesim-manv-public-test-scenarios) have been added~

### CODE REVIEW

**Comment by Greenscreen23** [comment]
I've refactored the radiogram selector to match the behavior and activity selectors. I've also omitted a changelog entry, as the difference between a radiogram model and a radiogram component has not been reflected in the changelog before.

File: frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-missing-transfer-connection/radigoram-card-content-missing-transfer-connection.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ng-container *ngIf="transferPointName$ | async as transferPointName">
+    <h5 class="card-title">Fehlende Verbindung</h5>
+    <span class="card-text"
+        >Es fehlt eine Verbindung zu diesem Transferpunkt: </span
+    ><span class="badge bg-secondary p-1">{{ transferPointName }}</span>
+</ng-container>
```
Comment (Nils1729) [inline frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-missing-transfer-connection/radigoram-card-content-missing-transfer-connection.component.html:6]: My understanding of the purpose of the radiogram is to communicate missing knowledge about the location in the domain. This text, however, looks more like an error message indicating a misconfigured exercise. It is an avoidable burden for  the signaler to translate it to the domain.

File: frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-missing-transfer-connection/radigoram-card-content-missing-transfer-connection.component.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ng-container *ngIf="transferPointName$ | async as transferPointName">
+    <h5 class="card-title">Fehlende Verbindung</h5>
+    <span class="card-text"
+        >Es fehlt eine Verbindung zu diesem Transferpunkt: </span
+    ><span class="badge bg-secondary p-1">{{ transferPointName }}</span>
```
Comment (Nils1729) [inline frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-missing-transfer-connection/radigoram-card-content-missing-transfer-connection.component.html:5]: Nowhere else do we use badges for transfer point names, rather for enum types.

File: frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-missing-transfer-connection/radigoram-card-content-missing-transfer-connection.component.html (hunk 1 above)
Comment (Greenscreen23) [reply frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-missing-transfer-connection/radigoram-card-content-missing-transfer-connection.component.html]: I've reworded it :)

**Review by Nils1729 — APPROVED** [review]

**Pull request merged**